lib/jekyll-github-metadata/site_github_munger.rb

```diff
@@ -55,7 +55,13 @@ def add_title_and_description_fallbacks!
           msg << "Jekyll GitHub Metadata will not set site.title to the repository's name."
           Jekyll::GitHubMetadata.log :warn, msg
         else
-          site.config["title"] ||= Value.new("title", proc { |_c, r| r.name })
+          site.config["title"] ||= Value.new("title", proc { |_c, r|
+            if r.project_page?
+              r.name # Someday if GitHub supports display names for repos, prefer it here.
+            else
+              r.owner_displayname || r.owner
+            end
+          })
```

Comment:
What do you think about expanding the variable names here? I understand that you're just rolling with what was already here, but I think using `_context` and `repository` instead of `_c` and `r` respectively would be a nice readable improvement.

Reply:
Sure, that sounds good. Should I change the entire file, for consistency? If so, I might want to submit a separate PR so as to not noise-up the diff.

Reply:
Since you're changing just this code here already, let's only do this part.